transportable-udfs-codegen/src/test/resources/outputs/sample-udf-properties/presto/sources/udfs/presto/OverloadedUDF10.java

```diff
@@ -0,0 +1,16 @@
+package udfs.presto;
+
+import com.linkedin.transport.api.udf.StdUDF;
+import com.linkedin.transport.presto.StdUdfWrapper;
+import java.lang.Override;
+
+public class OverloadedUDF10 extends StdUdfWrapper {
```

Comment:
Do we need 10 files (i.e., 10 overloadings) to test? If we can make the same point with less, let us reduce.